lib/Solver/FastCexSolver.cpp

```diff
@@ -260,6 +260,7 @@ class ValueRange {
   }
   
   std::int64_t minSigned(unsigned bits) const {
+    assert(bits >= 2 && bits <= 64);
```

Comment:
Can we do an `APInt::getSignedMinValue(bits).getSExtValue()` instead ?

Reply:
Yes, although not quite the way you might be hoping

Reply:
Implemented

Reply:
I'm not hoping - I'm just suggesting ;)